index.html

```diff
@@ -355,15 +353,18 @@
         position: relative;
         left: 80%;
         height: 380px;
-        top: -370px;
+        top: -320px;
     }
 
     .lets-work-together {
+        background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../../assets/images/spacebg.jpg");
```

Comment:
```suggestion
        background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../images/spacebg.jpg");
```

I think this path should work instead?

Reply:
Your proposed path does not, but perhaps this is what you were thinking `url("./assets/images/spacebg.jpg");`